### PR TITLE
add pangeo binder

### LINKED ISSUE #17: Add http://binder.pangeo.io/
Add http://binder.pangeo.io/ to the accepted hosts for bndr.it.

cc @@jhamman

### PR DESCRIPTION
**Pull request by jhamman**
closes #17

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -283,7 +283,9 @@ def __init__(self):
             db=db,
             prefixes={'mybinder': ['https://mybinder.org/'],
                       'gesis': ['https://notebooks.gesis.org/binder/'],
-                      'auto': ['https://notebooks.gesis.org/binder/',
+                      'pangeo': ['http://binder.pangeo.io'],
```
Comment (betatim) [inline app.py]: you don't want to make it https instead?

File: frontend/pages/index.js [hunk 2]
```diff
@@ -169,8 +170,9 @@ class Home extends React.Component {
         </p>
         <p>
           Currently only links to{" "}
-          <a href="https://mybinder.org">mybinder.org</a> and{' '}
-          <a href="https://notebooks.gesis.org/binder/">GESIS</a> are accepted.
+          <a href="https://mybinder.org">mybinder.org</a>, 
+          <a href="https://notebooks.gesis.org/binder/">GESIS</a> and{' '}
+          <a href="http://binder.pangeo.io">GESIS</a> are accepted.
```
Comment (betatim) [inline frontend/pages/index.js]: GESIS -> Pangeo or Pangeo Binder?

**Comment by betatim** [comment]
One day we will refactor this so a new destination doesn't need umpteen edits to add.

**Comment by jhamman** [comment]
Thanks @betatim!

**Pull request merged**